docs/QTIP.md

```diff
@@ -0,0 +1,29 @@
+# QTIP
+
+## What?
+
+QTIP is an MsQuic specific concept that allows MsQuic to exercise the QUIC protocol over XDP using TCP packets.
+The TCP packets here are a kind of trojan horse; we still leverage the QUIC protocol but masquerade it as TCP packets.
```

Comment:
An apt description, but I think people might get scared seeing the words "trojan horse" in official documentation as a product feature 😄 
Maybe the word "disguise" is less alarming?

Reply:
+1, I would limit the words with negative connotation like "trojan horse", "fool", "fake", "for cheap"...

What about:
"QTIP add a TCP header to the QUIC packet instead of a UDP header. It doesn't follow the TCP protocol after the initial handshake: headers are generated with good enough data to be compatible with most networks and leverage TCP specific optimizations, but are not used to operate the protocol.

You can think of it as disguising QUIC packets as TCP packets!"

Reply:
Noted. Pushed a fix :)